services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/AccessRequirementManagerImpl.java

```diff
@@ -263,32 +264,41 @@ public void deleteAccessRequirement(UserInfo userInfo,
 	}
 
 	static AccessRequirement setDefaultValues(AccessRequirement ar) {
-		if (!(ar instanceof ManagedACTAccessRequirement)) {
-			return ar;
-		}
-		ManagedACTAccessRequirement actAR = (ManagedACTAccessRequirement) ar;
-		if (actAR.getIsCertifiedUserRequired() == null) {
-			actAR.setIsCertifiedUserRequired(false);
-		}
-		if (actAR.getIsValidatedProfileRequired() == null) {
-			actAR.setIsValidatedProfileRequired(false);
-		}
-		if (actAR.getIsDUCRequired() == null) {
-			actAR.setIsDUCRequired(false);
-		}
-		if (actAR.getIsIRBApprovalRequired() == null) {
-			actAR.setIsIRBApprovalRequired(false);
-		}
-		if (actAR.getAreOtherAttachmentsRequired() == null) {
-			actAR.setAreOtherAttachmentsRequired(false);
-		}
-		if (actAR.getIsIDUPublic() == null) {
-			actAR.setIsIDUPublic(false);
-		}
-		if (actAR.getExpirationPeriod() == null) {
-			actAR.setExpirationPeriod(DEFAULT_EXPIRATION_PERIOD);
+		if (ar instanceof ManagedACTAccessRequirement) {
+			ManagedACTAccessRequirement actAR = (ManagedACTAccessRequirement) ar;
+			if (actAR.getIsCertifiedUserRequired() == null) {
+				actAR.setIsCertifiedUserRequired(false);
+			}
+			if (actAR.getIsValidatedProfileRequired() == null) {
+				actAR.setIsValidatedProfileRequired(false);
+			}
+			if (actAR.getIsDUCRequired() == null) {
+				actAR.setIsDUCRequired(false);
+			}
+			if (actAR.getIsIRBApprovalRequired() == null) {
+				actAR.setIsIRBApprovalRequired(false);
+			}
+			if (actAR.getAreOtherAttachmentsRequired() == null) {
+				actAR.setAreOtherAttachmentsRequired(false);
+			}
+			if (actAR.getIsIDUPublic() == null) {
+				actAR.setIsIDUPublic(false);
+			}
+			if (actAR.getExpirationPeriod() == null) {
+				actAR.setExpirationPeriod(DEFAULT_EXPIRATION_PERIOD);
+			}
+			return actAR;
+		} else if (ar instanceof SelfSignAccessRequirement) {
+			SelfSignAccessRequirement req = (SelfSignAccessRequirement) ar;
+			if (req.getIsCertifiedUserRequired() == null) {
+				req.setIsCertifiedUserRequired(false);
+			}
+			if (req.getIsValidatedProfileRequired() == null) {
+				req.setIsValidatedProfileRequired(false);
+			}
+			return req;
 		}
-		return actAR;
+		return ar;
```

Comment:
Since there is no overlap between these types each type should have its own method.